parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetFileWriter.java

```diff
@@ -425,6 +436,27 @@ public void testConvertToThriftStatistics() throws Exception {
   }
 
   @Test
+  public void testStatisticsGettingIgnoredFor160Version() throws Exception {
+    long[] longArray = new long[] {39L, 99L, 12L, 1000L, 65L, 542L, 2533461316L, -253346131996L, Long.MAX_VALUE, Long.MIN_VALUE};
+    LongStatistics parquetMRstats = new LongStatistics();
```

Comment:
I think we use empty ones for some reason. Probably fine for now, but I'll file a bug to fix it. That's a misleading API and could lead to incorrect results with predicate push-down.